tools/d2g/genericworker/generated_types.go

```diff
@@ -278,7 +278,6 @@ type (
 		// Maximum time the task container can run in seconds.
 		//
 		// Mininum:    1
-		// Maximum:    86400
```

Comment:
Would be nice to add a note to the description that the maximum value is variable, but worker-pool specific.